src/scribe_data/extract_transform/languages/English/formatted_data/translated_words.json

```diff
@@ -0,0 +1,849 @@
+            "sv": "dotter"
+        }
+    },
+    {
+        "son": {
+            "fr": "Fils",
+            "de": "Sohn",
+            "it": "Figlio",
+            "pt": "Filho",
+            "ru": "Сын",
+            "es": "hijo",
+            "sv": "Sonen"
+        }
+    },
+    {
+        "soup": {
+            "fr": "soupe",
+            "de": "Suppe",
+            "it": "La zuppa",
+            "pt": "Sopa",
+            "ru": "Суп",
+            "es": "Sopa",
+            "sv": "Sopp"
+        }
+    },
+    {
+        "space": {
+            "fr": "Espace",
+            "de": "Raum",
+            "it": "Spazio",
+            "pt": "Espaço",
+            "ru": "пространство",
+            "es": "Espacio",
+            "sv": "utrymme"
+        }
+    },
+    {
+        "car": {
+            "fr": "voiture",
+            "de": "Autos",
+            "it": "auto",
+            "pt": "carro",
+            "ru": "Автомобили",
+            "es": "El coche",
+            "sv": "Bilen"
+        }
+    },
+    {
+        "circle": {
+            "fr": "Cirque",
+            "de": "Kreis",
+            "it": "Circolo",
+            "pt": "Círculo",
+            "ru": "Круг",
+            "es": "Círculo",
+            "sv": "cirkel"
+        }
+    },
+    {
+        "sphere": {
+            "fr": "sphère",
+            "de": "Sphäre",
+            "it": "La sfera",
+            "pt": "Espécie",
+            "ru": "сфера",
+            "es": "Esfera",
+            "sv": "Sfera"
+        }
+    },
+    {
+        "steel": {
+            "fr": "Acier",
+            "de": "Stahl",
+            "it": "Acciaio",
+            "pt": "Aço",
+            "ru": "сталь",
+            "es": "El acero",
+            "sv": "stål"
+        }
+    },
+    {
+        "stomach": {
+            "fr": "Le ventre",
+            "de": "Magen",
+            "it": "dello stomaco",
+            "pt": "O estômago",
+            "ru": "желудок",
+            "es": "El estómago",
+            "sv": "magen"
+        }
+    },
+    {
+        "store": {
+            "fr": "Boutique",
+            "de": "Geschäfte",
+            "it": "negozio",
+            "pt": "Loja",
+            "ru": "магазин",
+            "es": "La tienda",
+            "sv": "Butiken"
+        }
+    },
+    {
+        "range": {
+            "fr": "Range",
+            "de": "Range",
+            "it": "Rango",
+            "pt": "Rango",
+            "ru": "Ранги",
+            "es": "Rango",
+            "sv": "Range"
+        }
+    },
+    {
+        "pig": {
+            "fr": "Le porc",
+            "de": "Schweine",
+            "it": "Il maiale",
+            "pt": "Porco",
+            "ru": "Свинья",
+            "es": "El cerdo",
+            "sv": "Svin"
+        }
+    },
+    {
+        "rice": {
+            "fr": "Le riz",
+            "de": "Riesen",
+            "it": "Il riso",
+            "pt": "O arroz",
+            "ru": "Рис",
+            "es": "El arroz",
```

Comment:
Happy to include it in here if @Linfye is comfortable with it :)

Reply:
> This could make sense, but maybe we could do a separate issue for it?

@andrewtavis I already have a branch in my local where i have moved the tranlations functions to `utils.py`, If you allow i can open a separate PR for this. or can open a PR for #77 russian translation process.
then other contributors can take reference from that.

Reply:
A separate PR for this sounds good, @shashank-iitbhu! From there we can jump to the Russian :) Appreciate you reaching out here!